WordPress/Classes/Utility/WPCrashlytics.m

```diff
@@ -0,0 +1,102 @@
+//
+//  WPCrashlytics.m
+//  WordPress
+//
+//  Created by Diego E. Rey Mendez on 3/26/15.
+//  Copyright (c) 2015 WordPress. All rights reserved.
+//
+
+#import "WPCrashlytics.h"
+
+#import <Crashlytics/Crashlytics.h>
+#import "AccountService.h"
+#import "BlogService.h"
+#import "ContextManager.h"
+#import "WPAccount.h"
+
+@interface WPCrashlytics () <CrashlyticsDelegate>
+@end
+
+@implementation WPCrashlytics
+
+#pragma mark - Dealloc
+
+- (void)dealloc
+{
+    [self stopObservingNotifications];
+}
+
+#pragma mark - Initialization
+
+- (instancetype)initWithAPIKey:(NSString*)apiKey
+{
+    NSParameterAssert(apiKey);
+    
+    self = [super init];
+    
+    if (self) {
+        [Crashlytics startWithAPIKey:apiKey];
+        [[Crashlytics sharedInstance] setDelegate:self];
+        
+        [self setCommonCrashlyticsParameters];
+        
+        [self startObservingNotifications];
+    }
+    
+    return self;
+}
+
+#pragma mark - Notifications
+
+- (void)startObservingNotifications
+{
+    NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
+    
+    [notificationCenter addObserver:self
+                           selector:@selector(handleDefaultAccountChangedNotification:)
+                               name:WPAccountDefaultWordPressComAccountChangedNotification
+                             object:nil];
+}
+
+- (void)stopObservingNotifications
+{
+    NSNotificationCenter *notificationCenter = [NSNotificationCenter defaultCenter];
+    
+    [notificationCenter removeObserver:self];
+}
+
+- (void)handleDefaultAccountChangedNotification:(NSNotification*)notification
+{
+    [self setCommonCrashlyticsParameters];
+}
+
+#pragma mark - Common crashlytics parameters
+
+- (void)setCommonCrashlyticsParameters
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
```

Comment:
Ha. Those were the exact three lines that were not expanded as I was looking at the PR.